app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: float,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
You should specify inner type of Car
```suggestion
    def serve_cars(self, cars: list[Car]) -> float:
```

Reply:
I did it, but I don't quite understand what it's for

Reply:
Specifying the inner type of a list in type hints makes your code more readable and easier for other developers to understand.